test/JsBuiltIns/ChakraLib.js

```diff
@@ -0,0 +1,17 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+    {
+        name: "Access to __chakraLibrary work",
+        body: function () {
+            assert.isNotUndefined(__chakraLibrary, "__chakraLibrary must NOT be undefined as we're using the -LdChakraLib flag.");

```

Comment:
With the LdChakraLib flag, overriding must not work. Test added. :)

---
In reply to: [115902571](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r115902571) [](ancestors = 115902571)